### PR TITLE
Use ESLint rules from eslint-config-girder

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
This causes testing to fail, due to incompatible versions of ESLint between this and Girder.

We'll have to wait until Girder is updated before merging.

**Comment by brianhelba** [comment]
This now works, and is rebased on master.

**Comment by brianhelba** [comment]
Going to rebase this to handle `plugin.cmake` conflicts.

**Comment by brianhelba** [comment]
Rebased.

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "backbone/no-silent": 2,
+      "backbone/no-view-model-attributes": 2,
+      "backbone/no-view-onoff-binding": 2,
+      "backbone/no-view-qualified-jquery": 2,
+      "backbone/render-return": 2
+    },
+    "globals": {
+        "isic": true
+    }
+}
```
Comment (msmolens) [inline .eslintrc]: It looks like we have to specify our custom bases to apply the backbone rules to our classes:
````
    ...
    "settings": {
        "backbone": {
            "Collection": [
                "Collection"
            ],
            "Model": [
                "Model"
            ],
            "View": [
                "View"
            ]
        }
    }
````

**Comment by brianhelba** [comment]
@msmolens Thanks, I forgot about that. Should be fixed now.

**Pull request merged**